tools/config.mk

```diff
@@ -30,6 +30,9 @@ ifeq ($(ARCH),)
   DISABLE_BACKUP?=0
   WOLFBOOT_VERSION?=0
   V?=0
+  LMS_LEVELS?=0
```

Comment:
These also need added to the list at the bottom

Reply:
done!